dev-libs/xapian/xapian-1.2.25.recipe

```diff
@@ -89,18 +101,31 @@ INSTALL()
 
 	rm -f "$libDir"/libxapian.la
 
+	if [ -n "secondaryArchSuffix" -a "$commandBinDir" != "$binDir" ]; then
+		mkdir -p "$binDir"
+		mv "$commandBinDir"/xapian-config "$binDir"
+	fi
+
 	prepareInstalledDevelLib libxapian
 	fixPkgconfig
 
-	mkdir -p $dataDir/cmake/Modules
-	mv $libDir/cmake/xapian $dataDir/cmake/Modules/xapian
+	# move $libDir/cmake/xapian/* to $dataDir/cmake/Modules/xapian/*
+	mkdir -p "$dataDir"/cmake
+	mv "$libDir"/cmake "$dataDir"/cmake/Modules
```

Comment:
I've only noticed this just now, but why was this moved?

Reply:
I don't know.
The recipe was created in 1439375f3ae420f8eb3a6ce82b0373db9d487ee4 with this move:
https://github.com/haikuports/haikuports/blob/1439375f3ae420f8eb3a6ce82b0373db9d487ee4/dev-libs/xapian/xapian-1.2.21.recipe#L82

If you think we should keep it in `$libDir` and if there are no downsides to this, then that's OK for me. 
(I can't change it myself because I'm not a cmake expert, so I might miss something.)